src/frontend/screens/ObservationDetails/TextArea.tsx

```diff
@@ -16,6 +15,7 @@ const TextArea = ({ value, field, onChange }: QuestionProps) => (
       multiline
       scrollEnabled={false}
       textContentType="none"
+      autoFocus={true}
```

Comment:
nit but the assignment here isn't necessary i.e.

```suggestion
      autoFocus
```